homeassistant/components/modbus/const.py

```diff
@@ -84,3 +93,11 @@
 }
 DEFAULT_TEMP_UNIT = "C"
 MODBUS_DOMAIN = "modbus"
+
+CONF_PLATFORMS = (
+    (CONF_CLIMATE, CONF_CLIMATES),
```

Comment:
```suggestion
    (CLIMATE_DOMAIN, CONF_CLIMATES),
```

Reply:
Please import the climate domain from the climate integration, etc.

Reply:
Done. I did not do this earlier, because I try to touch only the essential code in a PR, to make the review easier.